dart/5.Command_line_app/7_Getting_user_input.md

```diff
@@ -0,0 +1,64 @@
+# Getting _user input_
+
+Let's start by __changing the first thing the user will see__ in our program:
+
+```dart
+void main() {
+    print('Rock, paper or scissors? (r/p/s)');
+
+
+    //  Pseudo code
+    //  
+    //  As long as it is true
+    // Show the prompt to the user
+    // Read your response from the console
+    // If the input is valid ('r', 'p', 's')
+    // Execute computer random option
+    // Compare that move with the user's move
+    // Show the result
+    // If input is 'q
+    // Exit program
+    // If the input is another key
+    // Show 'Invalid input'
+}
+```
+
+After that, we need to get what the user enters and for that we are going to make use of a function inside a _package_, which is something we will use often.
+
+## _Packages_
+
+They are __libraries of functions and utilities that we can make use of, without having to write them ourselves__. To do this, we need to import these libraries.
+
+Let's begin by writing the function we need and Dart will help us to find out which library we are talking about. Write after the `print` the function `stdin;` (_stantard input_). You will see a red underline and next to it, Dart tells you what the error is: `Undefined name 'stdin'`. It's not recognizing that name in our code and that's because we haven't imported it yet.
+
+Hover the cursor over that name, and press the `cmd/ctrl + .` keys and a help menu will open. Once there, select `Import library dart:io`. Dart takes care of importing the library for us and you will see that there is no longer any failure:
+
+![Importando dart:io](https://raw.githubusercontent.com/themonkslab/courses/main/dart/5.app_linea_comando/5.7_importando_dart_io.gif)
```

Comment:
Didn't see the comment in the previous PR? It was about substitute of image being in spanish.

Maybe I didn't see it because once that the PR is closed is not that easy to get into the branch and check the comments... Maybe when it's something that should be addressed, is better to block it with requested changes.

In that case, I will do that with this one.